sockjs/session.py

```diff
@@ -399,5 +400,10 @@ def broadcast(self, message):
                 session.send_frame(blob)
 
     def __del__(self):
-        self.clear()
+        if len(self.sessions):
+            warnings.warn(
+                "Unclosed sessions! "
+                "Please call await clear() before del",
```

Comment:
backticks around `await cliear()` make the message more readable.

Reply:
Made better warning message.